dev-scripts/build-local.sh

```diff
@@ -56,4 +57,9 @@ cp "${K3S_AIRGAP_IMAGES_TARBALL}" "${PACKAGE_FOLDER}"
 
 DOCKERFILE="${SCRIPT_DIR}/../package/Dockerfile"
 # Always use buildx to make sure the image & the binary architectures match
-docker buildx build -t "${TARGET_REPO}" -f "${DOCKERFILE}" "${PACKAGE_FOLDER}" --platform="${TARGET_OS}/${TARGET_ARCH}"
+docker buildx build -t "${TARGET_REPO}" -f "${DOCKERFILE}" \
```

Comment:
Doesn't this mean we now need to add `--build-arg` whenever we add a field to the `build.yaml`? We should script this to infer the `--build-arg` additions that need to be added by reading the `build.yaml` source (or `build.sh`, if we use the suggestion I have in my review above).

Reply:
Not all values are fit for the Dockerfile. We are selective about which go there and which end up referenced as constants in regular Go code. Thus I'd rather not assume that the Dockerfile should include **all** values as ENV values.

Reply:
We should document this.

Reply:
We can't document the intended use of each value in advance because we can't know who might want to use one and where. The good thing is that the generated constants file has all of them, whether one plans to use some variable in Go code or not.